adele-booking/src/main/java/com/microservicesteam/adele/booking/boundary/web/BookingController.java

```diff
@@ -1,17 +1,13 @@
 package com.microservicesteam.adele.booking.boundary.web;
 
+import java.util.List;
+
+import org.springframework.web.bind.annotation.*;
```

Comment:
Please set star import count to `100` to avoid it (Settings > Editor > Code Style > Java > Imports tab)